sonar-batch-protocol/src/main/gen-java/org/sonar/server/source/db/FileSourceDb.java

```diff
@@ -542,6 +555,7 @@ public boolean hasScmRevision() {
       }
     }
 
+    // optional string scm_author = 4;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)